### PR TITLE
production only classification heartbeat warning

### PR DESCRIPTION
**Pull request by camallen**
don't cry wolf in staging system, provide better copy with rails env and only run in staging. Happy to change the recipient list from a yaml config to env vars if peeps want.

Describe your change here.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by marten** [comment]
Should the recipient just be one of those mailing lists I keep hearing good things about?

**Comment by camallen** [comment]
Indeed - it's using env vars now that have been setup on prod / staging with a list email address.

File: app/views/classification_heartbeat_mailer/missing_classifications.text.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Alert: all hands on deck!
+Team Alert!
 
-Eeek! No classification data received via panoptes in the last <%= (@window_period / 60).to_i %> minutes!
+No classification data received in the last <%= (@window_period / 60).to_i %> minutes!
```
Comment (marten) [inline app/views/classification_heartbeat_mailer/missing_classifications.text.erb]: How will that work with the new default of 15 seconds? "0 minutes"?

Comment (camallen) [inline app/views/classification_heartbeat_mailer/missing_classifications.text.erb]: good point, i'll update the default to something sensible.

**Pull request merged**